Documentation/FAQ.md

```diff
@@ -28,11 +28,15 @@ The SerenityOS project tries to maximize hackability, accountability, and fun(!)
 
 ## Does SerenityOS have a package manager?
 
-No, SerenityOS does not have a package manager. The project uses a monorepo approach, meaning that all software is built in the same style and using the same tools. There is no reason to have something like a package manager because of this.
+To simplify: No, SerenityOS does not have a "Linux-style" package manager with pre-built software. The `pkg` tool interacts with the existing SerenityOS Ports system.
 
-*However* there are ports which can be found in the [Ports directory](../Ports). A port is a piece of software that can optionally be installed, might not have been built by us, but supports running on SerenityOS. They act quite similarly to packages, each coming with their own install script.
+More specifically: SerenityOS does not have a package manager that allows you to download pre-built software. The project uses a monorepo approach, meaning that all software is built in the same style and using the same tools. Additionally, the SerenityOS ABI, such as library symbols or syscall interfaces, have absolutely no stability guarantee and should be expected to change at any moment. There is no reason to have a binary package manager because of this.
```

Comment:
replace colon with comma?

We should include a mention that "LibC is the ABI, similar to OpenBSD's guarantees" or some such. (assuming our guarantees are actually similar to OpenBSD, but I remember that being the assumption from discussions of porting the Go language). Though I see you mentioned that library symbols are not stable either, maybe that's safer to promise here.

Reply:
You're right, the LibC ABI thing should be mentioned; it also came up with Zig.